travelwise/travelwise/settings.py

```diff
@@ -33,7 +33,7 @@
 CSRF_TRUSTED_ORIGINS = os.environ.get("DJANGO_CSRF_TRUSTED_ORIGINS").split(" ")
 
 # HTTPS Redirect
-SECURE_SSL_REDIRECT = not DEBUG
+#SECURE_SSL_REDIRECT = not DEBUG
```

Comment:
We need to fix some settings for this, since its going into a recursive loop when pushed for production

Reply:
What kind of loop?

Reply:
I need to set the SECURE_PROXY_SSL_HEADER since I'm running it behind a proxy.

https://docs.djangoproject.com/en/4.2/ref/settings/#secure-ssl-redirect